tools/pipeline-witness/Azure.Sdk.Tools.PipelineWitness.Tests/UnitTest1.cs

```diff
@@ -1,8 +1,7 @@
-using System;
-using Xunit;
-
 namespace Azure.Sdk.Tools.PipelineWitness.Tests
 {
+    using Xunit;
```

Comment:
Is this the default style? Most code bases I've worked in keep them at the top of the file and not nested under namespace.

Reply:
There's a functional difference between inside and outside.  With using declared outside a namespace, code in the namespace will prefer types defined in the namespace over those from the usings.  The example I've seen is by defining a class called Math in Azure.Sdk.Tools causing compile time breaks in every class that puts `using Sytem;` outside their namespace.   With usings inside the namespace, types defined in the external namespaces take precedence over types defined in the namespace.

The default for azure-sdk-for-net and dotnet/runtime is usings outside of namespaces:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Operation.cs

Reply:
I moved them back outside the namespace for this PR